gini/gini-vision-lib-ios
# Update documentation for file import.

Added documentation for file import and open with. 
Moved all Sphinx guides into Jazzy and added a custom theme for this.

### How to test
Run build-documentation-api.sh script and see how it looks (Documentation/API/index.html)

### Merging
Automatic.

### Ticket
#60
